src/pages/Bridge/BurnForm/index.tsx

```diff
@@ -141,7 +141,7 @@ const BurnForm = (): JSX.Element | null => {
       }
     };
 
-    const validateForm = (value: number): string | undefined => {
+    const validateForm = (value: string): string | undefined => {
```

Comment:
There are a couple of these changes - this was an uncaught error. The values were already being passed as strings, and for some reason the type error wasn't being caught.